assets/javascripts/discourse/connectors/post-text-buttons/ai-post-helper-trigger.gjs

```diff
@@ -133,65 +115,44 @@ export default class AiPostHelperTrigger extends Component {
 
   @action
   async showAiPostHelperMenu() {
-    this.highlightSelectedText();
-    if (this.site.mobileView) {
-      this.currentMenu.close();
-
-      await this.menu.show(virtualElementFromTextRange(), {
-        identifier: "ai-post-helper-menu",
-        component: AiPostHelperMenu,
-        inline: true,
-        interactive: true,
-        placement: this.shouldRenderUnder ? "bottom-start" : "top-start",
```

Comment:
`this.shouldRenderUnder` doesn't exist here, it's seems like it's a copy pasta error